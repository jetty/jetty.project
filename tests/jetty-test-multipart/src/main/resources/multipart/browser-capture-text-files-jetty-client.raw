--JettyHttpClientBoundary1e87p8a551psw1al
Content-Disposition: form-data; name="text"
Content-Type: text/plain;charset=UTF-8

text default

--JettyHttpClientBoundary1e87p8a551psw1al
Content-Disposition: form-data; name="file1"; filename="a.txt"
Content-Type: application/octet-stream
X-SHA1: 588A0F273CB5AFE9C8D76DD081812E672F2061E2

Content of a.txt

--JettyHttpClientBoundary1e87p8a551psw1al
Content-Disposition: form-data; name="file2"; filename="a.html"
Content-Type: application/octet-stream
X-SHA1: 9A9005159AB90A6D2D9BACB5414EFE932F0CED85

<!DOCTYPE html><title>Content of a.html.</title>
--JettyHttpClientBoundary1e87p8a551psw1al--
