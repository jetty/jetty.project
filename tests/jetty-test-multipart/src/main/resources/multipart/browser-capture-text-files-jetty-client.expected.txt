Request-Header|Accept-Encoding|gzip
Request-Header|Connection|close
Request-Header|Content-Type|multipart/form-data; boundary=JettyHttpClientBoundary1e87p8a551psw1al
Request-Header|Host|localhost:9090
Request-Header|Transfer-Encoding|chunked
Request-Header|User-Agent|Jetty/9.4.9.v20180320
Request-Header|X-BrowserId|jetty-client
Parts-Count|3
Part-ContainsContents|text|text default
Part-ContainsContents|file1|Content of a.txt
Part-ContainsContents|file2|<!DOCTYPE html><title>Content of a.html
Part-Filename|file1|a.txt
Part-Filename|file2|a.html
Part-Sha1sum|file1|588A0F273CB5AFE9C8D76DD081812E672F2061E2
Part-Sha1sum|file2|9A9005159AB90A6D2D9BACB5414EFE932F0CED85